Code chunk #1

```diff
@@ -0,0 +1,443 @@
+
+      editBttn.addEventListener('click', () => {
+        Methods.editTask(taskLabel, editBttn, doneBttn);
+      });
+      doneBttn.addEventListener('click', () => {
+        // eslint-disable-next-line no-use-before-define
+        this.editLocalSt(editBttn, doneBttn, taskLabel, taskWrap.id);
+      });
+
+      moreBttn.addEventListener('click', () => {
+        moreMenu.classList.replace('menuHide', 'menuVis');
+        moreBttn.classList.replace('menuVis', 'menuHide');
+      });
+      closeMenu.addEventListener('click', () => {
+        moreBttn.classList.replace('menuHide', 'menuVis');
+        moreMenu.classList.replace('menuVis', 'menuHide');
+      });
+
+      checkbox.addEventListener('click', (e) => {
+        // eslint-disable-next-line no-use-before-define
+        this.taskStatusModifier(e);
+      });
+
+      deleteBttn.addEventListener('click', (e) => {
+        // eslint-disable-next-line no-use-before-define
+        this.deleteTask(e);
+      });
+
+      taskWrap.append(moreBttn);
+    }
+  }
+
+  getTaskName(taskInput) {
+    const { tasksArr } = this;
+    const newTask = new Task(taskInput.value);
+    tasksArr.push(newTask);
+    taskInput.value = '';
+  }
+
+  addToLocalStorage() {
+    const { tasksArr } = this;
+    const arrLocalStorage = JSON.stringify(tasksArr);
+    localStorage.setItem('tasks', arrLocalStorage);
+  }
+
+  static addAsObject(tasksArr) {
+    const arrLocalStorage = JSON.stringify(tasksArr);
+    localStorage.setItem('objeto', arrLocalStorage);
+  }
+
+  setIndex() {
+    const { tasksArr } = this;
+    for (let i = 0; i < tasksArr.length; i += 1) {
+      tasksArr[i].index = i + 1;
+    }
+  }
+
+  taskStatusModifier(e) {
+    const { tasksArr } = this;
+    const box = e.target.checked;
+    const div = e.target.parentNode.childNodes[1];
+    const id = e.target.parentNode.id - 1;
+    if (box === true) {
+      div.classList.add('completed');
+      tasksArr[id].completed = true;
+      this.addToLocalStorage();
+    }
+    if (box === false) {
+      div.classList.remove('completed');
+      tasksArr[id].completed = false;
+      this.addToLocalStorage();
+    }
+  }
+
+  editLocalSt(edBttn, dnBttn, label, currentIndex) {
+    const { tasksArr } = this;
+    edBttn.classList.replace('menuHide', 'menuVis');
+    dnBttn.classList.replace('menuVis', 'menuHide');
+    label.readOnly = true;
+
+    for (let i = 0; i < tasksArr.length; i += 1) {
+      if ((currentIndex - 1) === i) {
+        tasksArr[i].description = label.value;
+      }
+    }
+    this.addToLocalStorage();
+  }
+
+  deleteTask(e) {
+    const { tasksArr } = this;
+    listSection.innerHTML = '';
+    const taskID = e.target.parentNode.parentNode.id - 1;
+
+    for (let i = 0; i < tasksArr.length; i += 1) {
+      if (taskID === i) {
+        tasksArr.splice(i, 1);
+      }
+    }
+
+    this.setIndex();
+    this.addToLocalStorage();
+    this.createHTML();
+  }
+
+  clearTasks() {
```

Comment:
code looks great :) 

but I think it could be better to use the filter method instead of the for loop to filter the array because it uses less code and is a little more readable.